Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,35 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            var flag = true;  
+            var temp = 0 ;
+            var numLength = 0; 
+
+            if (array == null)
+            {
+                numLength = 1; 
+            }
+            else
+            {
+                numLength = array.Length; 
+            }
+
+           //sorting an array  
+            for (var i = 1; (i <= (numLength - 1)) && flag; i++)  
+            {  
+                flag = false;  
+                for (var j = 0; j < (numLength - 1); j++)  
+                {  
+                    if (array[j + 1] < array[j])  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['array' is null on at least one execution path.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887669)